0000-rep-template.md

```diff
@@ -0,0 +1,45 @@
+---
+REP Number: <Fill me in with a four-digit unique incrementing number, matching the pull request number; update AFTER REP acceptance.>
+Author: <Author's First Name and Last Name.>
+Start Date: <Today's Date, YYYY-MM-DD>
+Bicep Issue Number: <Fill if the REP originates from a Bicep issue; otherwise, delete.>
+Enhances: <Fill an existing final REP number if this REP enhances it; otherwise, delete.>
+Deprecates: <Fill an existing final REP number if this REP deprecates it; otherwise, delete.>
+Enhanced By: <Applicable to existing final REPs only. Fill a final REP number if that REP enhances this one. Delete for new REPs.>
+Deprecated By: <Applicable to existing final REPs only. Fill a final REP number if that REP deprecates this one. Delete for new REPs.>
+---
+
+# Title - replace me with feature name
+
+## Summary
+
+Provide a brief paragraph explaining the feature.
+
+## Motivation
+
+Outline the reasons behind implementing this feature, the use cases it supports, and the expected outcomes.
```

Comment:
Add [Terms and definitions](https://github.com/radius-project/design-notes/blob/main/template/YYYY-MM-design-template.md#terms-and-definitions).